### ISSUE TITLE
False Positive on cgo "export" comments

### ISSUE THREAD

**Issue by RevenantBob** [golang/lint#462]
Using cgo a function can be exported with:
`
//export FunctionName

func FunctionName() {
...
}
`
golint is reporting the comment malformatted and suggesting the comment be extended with a description. This would break compilation.

**Comment by ianlancetaylor** [comment]
What golint is expecting is something like

```Go
// FunctionName is used to name the function.
//export FunctionName
func FunctionName() {
    ...
}
```

**Comment by RevenantBob** [comment]
Maybe a little more verbose on that, but you are correct!
